### PR TITLE
LPS-153817 Add Text Input fragment

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"dataType": "string",
+					"defaultValue": "Type Something",
```
Comment (p2kmgcl) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json:7]: Two cosiñas about this:
- Shouldn't it be localizable?
- Maybe a default value of "" is more useful :sweat_smile:

**Review by p2kmgcl — COMMENTED** [review]
This is very bonico

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json:7]: I saw that value in the figma and I thought it was the default one 😅

Comment (sandrodw3) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json]: Updated both things 🥰

Comment (sandrodw3) [reply modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-inputs/src/main/resources/com/liferay/fragment/collection/contributor/inputs/dependencies/text-input/index.json]: Would you take a look?

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**